File: tests/project_multilang/swift/input/Caller.swift
func run(_ c: Any) {
  (c as! ICallback).onReady()
  if (c as! ICallback).isEnabled() {
    print("on")
  }
}

func multiline(flag: Bool) {
  let isOneWay =
      !INTL_FLAG
          && flag
          && other()
  use(isOneWay)
}

func other() -> Bool { return true }
func use(_ b: Bool) { print(b) }

File: tests/project_multilang/kotlin/input/Caller.kt
package demo
fun run(c: Any) {
  (c as ICallback).onReady()
  if ((c as ICallback).isEnabled()) {
    println("on")
  }
}

fun multiline(flag: Boolean) {
  val isOneWay =
      !FEATURE_FLAG
          && flag
          && other()
  use(isOneWay)
}

fun other(): Boolean = true
fun use(b: Boolean) { println(b) }
